_extensions/acronyms/acronyms_styles.lua

```diff
@@ -35,82 +35,81 @@
 local Helpers = require("acronyms_helpers")
 
 
-local function capitalize_first(s)
-  return (s:gsub("^%l", string.upper))
-end
-
 -- The table containing all styles, indexed by the style's name.
 local styles = {}
 
 
 -- Local helper function to create either a Str or a Link,
 -- depending on whether we want to insert links.
-local function create_element(content, key, insert_links)
+-- Kept as a local reference here to keep compatibility with previous versions.
+create_element = Helpers.create_rich_element
```

Comment:
I'm not sure we need to keep this; we can simply replace the reference in all styles, and update the documentation.